adal/build/Microsoft.IdentityModel.Clients.ActiveDirectory.nuspec

```diff
@@ -32,15 +32,17 @@ ADAL.NET takes advantage of Windows Server Active Directory and Azure Active Dir
         <dependency id="System.Runtime.Serialization.Json" version="4.3.0" exclude="Build,Analyzers" />
         <dependency id="System.Runtime.Serialization.Primitives" version="4.3.0" exclude="Build,Analyzers" />
         <dependency id="System.Security.Cryptography.X509Certificates" version="4.3.0" exclude="Build,Analyzers" />
-        <dependency id="System.Xml.XDocument" version="4.3.0" exclude="Build,Analyzers" />
+        <dependency id="System.Xml.XDocument" version="4.3.0" exclude="Build,Analyzers" />        
       </group>
       <group targetFramework="MonoAndroid8.1">
```

Comment:
This is the actual fix - since we bundle core.dll, we need to copy it's dependencies.